.gitignore

```diff
@@ -3,3 +3,4 @@
 .idea
 node_modules
 .next
+public
```

Comment:
Don't include public here. The content inside is not generated.

Reply:
ah, it is also a leftover from the old website 🤦